include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
i have create simple console application in vs2015,

and copy libuv.lib and header files to application source directory.

try to compile console application but show some compile error,
so i have to rebuild libuv.lib with my fix, that's all.

the source code is simple:

int main()
{
  uv_loop_t *loop = (uv_loop_t *)malloc(sizeof(uv_loop_t));
  uv_loop_init(loop);
  printf("Now quitting.\n");
  uv_run(loop, UV_RUN_DEFAULT);
  uv_loop_close(loop);
  free(loop);
  return 0;
}